### PR TITLE
'Pollute' command

### PR DESCRIPTION
**Pull request by joe513**
#2 Fixation

### CODE REVIEW

File: aliexpress/utils.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import random
+from string import ascii_letters
+import django.utils.timezone as t
```
Comment (inoks) [inline aliexpress/utils.py]: Please avoid using such short names if possible. Here it's just fine to use: `from django.utils import timezone`

File: aliexpress/management/commands/pollute.py [hunk 2]
```diff
@@ -8,16 +11,42 @@ class Command(BaseCommand):  # Simple command for data polluting
...
+        for categ in category_objects:
+            categ.save()
+
+            choosen_rate = random.choice(rate_objects)
+            choosen_rate.save()
+
+            product_objects = models.Product.objects.bulk_create(
+                [models.Product(name=string_generator(12), description=string_generator(54),
+                                    rate=choosen_rate, category=categ) for p in range(PRODUCTS_NUMBER)])
+            for prod in product_objects:
```
Comment (inoks) [inline aliexpress/management/commands/pollute.py]: Is it really required to call extra `.save()` here?

Comment (joe513) [reply aliexpress/management/commands/pollute.py]: Yes, it is, bulk_create doesn't call `.save()` method

Comment (inoks) [reply aliexpress/management/commands/pollute.py]: do you really need to call `save()` method? I don't see any `save()` method defined on model.

Comment (joe513) [reply aliexpress/management/commands/pollute.py]: Try to delete the `.save` method and run the program, you will gen an error.

Comment (inoks) [reply aliexpress/management/commands/pollute.py]: Yes, you right, but that does not mean objects was not created at that moment.
So you can't access it iterating with `for prod in product_objects:` but should use smth like: `categ.product_set.all()`.

**Pull request merged**